docs/users_guide/community_index.md

```diff
@@ -48,25 +46,67 @@ In addition to the following information, The Carpentries also maintains data on
 
 Individuals can view this information but can not edit it directly.  Please contact team@carpentries.org with questions about this information.
 
-### Instructor Training
+## Instructor Training
+
+Carpentries Instructor trainees can log in to AMY to view their checkout progress towards their Instructor badge and submit their "Get Involved" step towards certification.  More information about the [Instructor Training checkout process is in our curriculum](https://carpentries.github.io/instructor-training/checkout.html).  Log into your AMY profile and click on "Training Progress" in the top menu bar:
+
```

Comment:
You could maybe link to the section at the top called Logging in

Reply:
Addressed with https://github.com/carpentries/amy/pull/2509/commits/d4ece899049456c41f94baa1098f80ed0684d448